app/src/main/java/me/ivg2/foodapp/RecipeFragment.java

```diff
@@ -86,6 +95,59 @@ public void onClick(View view) {
         rvRecipes.setLayoutManager(new LinearLayoutManager(getActivity()));
         rvRecipes.setAdapter(recipeAdapter);
 
+        tabs = view.findViewById(R.id.tab_host);
+        tabs.setSelectedTabIndicatorColor(getResources().getColor(R.color.colorAccent));
+        tabs.setTabTextColors(Color.parseColor("#000000"),getResources().getColor(R.color.colorAccent));
+        tabs.addOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
+            @Override
+            public void onTabSelected(TabLayout.Tab tab) {
+                refreshRecommendedRecipes();
+
+                switch (tab.getPosition()) {
+                    case 0:
```

Comment:
not a big fan of case 0 and case 1: make a const called RECIPES_TO_COOK_POSITION = 0and RECIPES_CLOSE_TO_COOKING = 1, or something that is more clear.